src/stylesheets/components/_example.scss

```diff
@@ -2,13 +2,12 @@
 .app-example-wrapper {
   @include govuk-responsive-margin(6, "top");
   @include govuk-responsive-margin(6, "bottom");
-  @include govuk-font(19);
 }
 
 .app-example {
   position: relative;
   border: 1px solid $govuk-border-colour;
-  background: govuk-colour("light-grey");
+  background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAJUlEQVQoU2P88ePHfwY0wMHBwYguxjgUFKI7GsTH5m4M3w1ChQAZTSeO0/AZpgAAAABJRU5ErkJggg==") repeat;
```

Comment:
It might be good to have a code comment here to say what this affects as it might not be obvious.